test/remoting-coercion.test.js

```diff
@@ -12,7 +12,12 @@ describe('remoting coercion', function() {
     var app = loopback();
     app.use(loopback.rest());
 
-    var TestModel = app.model('TestModel', { base: 'Model', dataSource: null, public: true });
+    var TestModel = app.registry.createModel('TestModel',
+      {},
+      { base: 'Model', public: true }
```

Comment:
`public` should be passed to `app.model`, not `createModel`.

Here is the rule to make this easier to remember: `createModel()` takes settings from `common/models/my-model.json`, `app.model` takes settings from `server/model-config.json`.

```js
var TestModel = app.registry.createModel('TestModel',
  {},
  { base: 'Model' });
app.model(TestModel, { public: true });
```

IIRC, models are public by default, which would explains why the test works this way too.